tests/foreman/api/test_docker.py

```diff
@@ -1132,7 +1132,7 @@ def setUpClass(cls):
         cls.cr_external = entities.DockerComputeResource(
             name=gen_string('alpha'),
             organization=[cls.org],
-            url=settings.docker.external_url,
+            url='http://{0}:2375'.format(settings.server.hostname),
```

Comment:
1) Makes sense
2) By default it is set to 'http://localhost:2375' which is basically the same as you're proposing. So are you suggesting to fix the issue on Jenkins properties file instead of tests?